solutions/instances/main.tf

```diff
@@ -57,6 +57,20 @@ locals {
   ) : null
 
   kms_region = (length(local.bucket_config_map) != 0) ? (var.existing_cos_kms_key_crn == null ? element(split(":", var.existing_kms_instance_crn), length(split(":", var.existing_kms_instance_crn)) - 5) : null) : null
+  at_cos_route = [{
+    route_name = "at-cos-route"
```

Comment:
I suggest you add a `var.enable_at_event_routing_to_cos` variable here too (default it to true)

Reply:
added new variable